src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -1388,6 +1414,242 @@ public async Task TestSchemaIntrospectionQuery(bool enableIntrospection, bool ex
             }
         }
 
+        /// <summary>
+        /// Test different graphql endpoints in different host modes
+        /// when accessed interactively via browser.
+        /// </summary>
+        /// <param name="endpoint">The endpoint route</param>
+        /// <param name="hostModeType">The mode in which the service is executing.</param>
+        /// <param name="expectedStatusCode">Expected Status Code.</param>
+        /// <param name="expectedContent">The expected phrase in the response body.</param>
+        [DataTestMethod]
+        [TestCategory(TestCategory.MSSQL)]
+        [DataRow("/swagger", HostModeType.Development, HttpStatusCode.OK, "", DisplayName = "SwaggerUI enabled in development mode.")]
+        [DataRow("/swagger", HostModeType.Production, HttpStatusCode.BadRequest, "", DisplayName = "SwaggerUI disabled in production mode.")]
+        public async Task OpenApi_InteractiveSwaggerUI(
+            string endpoint,
+            HostModeType hostModeType,
+            HttpStatusCode expectedStatusCode,
+            string expectedContent = "")
```

Comment:
nit: is it possible to use string.empty here?

Reply:
updated to use default values. added comments.